src/integrations/third-party/elementor.php

```diff
@@ -572,6 +575,14 @@ protected function get_replace_vars() {
 			'user_description',
 			'pagetotal',
 			'pagenumber',
+			'post_year',
+			'post_month',
+			'post_day',
+			'author_first_name',
+			'author_last_name',
+			'permalink',
+			'postcontent',
```

Comment:
This has been renamed to `post_content`.

Reply:
👍 . It has been renamed to `post_content` in the `elementor.php` file.

Reply:
See this commit: [`01ecd19` (#17430)](https://github.com/Yoast/wordpress-seo/pull/17430/commits/01ecd19320ff39b663d26d1bdc5563df87f726a3)